node/node.go

```diff
@@ -123,7 +123,8 @@ func New(cfg *Config, version string) (*Node, error) { //nolint:gocyclo,funlen
 
 	services := make([]service.Service, 0)
 
-	chain := blockchain.New(database, &cfg.Network)
+	var synchronizer *sync.Synchronizer
+	chain := blockchain.New(database, &cfg.Network, synchronizer.PendingBlock)
```

Comment:
it causes nil pointer dereference because `synchronizer` is not set before func pointer is passed (`synchronizer.PendingBlock`). Tbh I'm quite suprise that go allows to pass func pointer of variable that is nil

Reply:
fixed